jenkins-examples/pipeline-examples/declarative-examples/build-trigger-example/Jenkinsfile

```diff
@@ -1,26 +1,32 @@
 pipeline {
     agent any
 
+    environment {
+        // The URL of the artifact in Artifactory that caused the build.
+        // May be empty if the build isn't been triggered by a change in Artifactory.
```

Comment:
Let's rephrase the description a bit (same goes for the other example):
// The URL of the artifact in Artifactory, that caused the job to be triggered. 
 // May be empty if the build isn't triggered by a change in Artifactory.